### ISSUE TITLE
Seeing log messages from timbre during compilation

### ISSUE THREAD

**Issue by cross** [ring-clojure/ring#279]
In a project I'm working on, built by leiningen, I am seeing a message during compilation of one of the clj files/classes.  The message is from timbre, it seems, and I'd assumed it was something we were doing wrong.  (We have recently begun using timbre for logging in this application)
```
>>>>>> lein uberjar
Compiling foobar.elector
Compiling foobar.module
Compiling foobar.rest-server
17-02-15 20:55:44 ubuntu-dev DEBUG [org.eclipse.jetty.util.log:89] - Logging to com.github.fzakaria.slf4j.timbre.TimbreLoggerAdapter@2e883c86 via org.eclipse.jetty.util.log.Slf4jLog
Compiling foobar.core
Compiling foobar.ambari-server-ha
Compiling foobar.schedule-operator
Compiling foobar.operator
Compiling foobar.fxtunnel
Created /data/cross2/tool/foobar/target/foobar-0.1.0-SNAPSHOT.jar
Created /data/cross2/tool/foobar/target/foobar-0.1.0-SNAPSHOT-standalone.jar
```

But in trying to figure out if our code was doing something to cause this, I've reduced rest_server.clj down to merely:
```
(ns foobar.rest-server
  "REST server module for Foobar"
  (:require
    [ring.adapter.jetty :refer [run-jetty]]))
```

That produces the above message during compilation.  Is there anything we're doing wrong to cause this?  Do you know why some timbre logging code would be running during compilation requiring ring.adapter.jetty?

**Comment by weavejester** [comment]
Jetty is pretty noisy when you load it, so just the act of using the classes produces a log message through slf4j, which in your case is then routed through Timbre. You should be able to turn it off by setting the `org.eclipse.jetty.util.log.announce` property to `false`, though I haven't tried it.

**Comment by cross** [comment]
Okay.  Thank you.  I'm trying to figure out how/where to set this.  I am not a clojure person, and am just trying to work around this oddity while other members of the team work on real issues.

Putting it in rest_server.clj above the (ns), making:
```
(System/setProperty "org.eclipse.jetty.util.log.announce" "false" )
(ns roboadmin.rest-server
  "REST server module for Tinderbox RoboAdmin"
  (:require
    [ring.adapter.jetty :refer [run-jetty]]))
```
doesn't work.  Trying to use a boolean "false" instead of a string, it fails to compile trying to cast it, much as would be expected.  Let me know if you know of a way to set this, or if this means this suggestion doesn't work.

**Comment by weavejester** [comment]
Try putting a file in your `resources` directory called `jetty-logging.properties` with the following content:

```
org.eclipse.jetty.util.log.announce=false
```

**Comment by weavejester** [comment]
Alternatively, add a `:jvm-opts` key to your project.clj file and set the property that way:

```clojure
:jvm-opts ["-Dorg.eclipse.jetty.util.log.announce=false"]
```

**Comment by cross** [comment]
I know I tried your second suggestion with no effect.  That was one of my first thoughts to implement your suggestion.  I'll try to find a place to put a properties file and see if that works.  Let me know if you have any other suggestions...  Thanks.

**Comment by cross** [comment]
Verified that the `:jvm-opts` in project.clj doesn't seem to help.  I tried adding it in both the root of the project, and within `:profiles { :uberjar {}}` with no effect.
However, I find that creating the described `jetty-logging.properties` in a resources directory _does_ work.  So thank you for that help.  I wasn't sure at first if I wanted this to affect the program when running in production, but my team tells me that the announcement ended up on stderr, instead of in the log file configured at runtime, so it's good to be gone.  Thanks!
